app/src/main/java/com/sun/chat_04/ui/login/LoginContract.kt

```diff
@@ -0,0 +1,17 @@
+package com.sun.chat_04.ui.login
+
+import com.facebook.AccessToken
+import com.sun.chat_04.ui.BasePresenter
+import com.sun.chat_04.ui.BaseView
+
+interface LoginContract {
+    interface View : BaseView {
+        fun onLoginListener(isSuccessful: Boolean)
```

Comment:
```suggestion
        fun onLoginSuccess(value: Any.....)
```
em nên viết cụ thể ra ntn
login success thì trả về giá trị gì
login error thì trả về giá trị gì?